invenio_communities/communities/services/config.py

```diff
@@ -106,6 +106,7 @@ class CommunityServiceConfig(RecordServiceConfig):
     }
 
     links_search = pagination_links("{+api}/communities{?args*}")
+    links_featured_search = pagination_links("{+api}/communities/featured{?args*}")
```

Comment:
since we are adding new things to the payload, maybe it makes sense to update the docs: https://inveniordm.docs.cern.ch/reference/rest_api_communities/#search-communities

Reply:
This will only add links to the search of featured communities:
https://inveniordm.docs.cern.ch/reference/rest_api_communities/#search-featured-communities